bitrise.yml

```diff
@@ -19,6 +19,15 @@ workflows:
         inputs:
         - content: go test -p 1 $BITRISE_GO_PACKAGES
 
+  kotlin_test:
```

Comment:
Please add an integration test under tests/_integration.
Please add a new fodler with a sample Kotlin step kotlin_toolkit_step_template that contains an example Step, so no reference to external repository is needed.